README.md

```diff
@@ -292,6 +292,36 @@ $builder = UnleashBuilder::create()
     ]);
 ```
 
+
+## Proxy SDK
+
+By default the SDK uses the Server-side endpoints on the Unleash API. You can also use the Proxy SDK, which is a
+lightweight SDK that uses the Client-side endpoints on the Unleash API. The Proxy SDK give a substantial performance improvement when using a large set of feature toggles (10K+).

```

Comment:
> By default the SDK uses the Server-side endpoints on the Unleash API. You can also use the Proxy SDK, which is a lightweight SDK that uses the Client-side endpoints on the Unleash API

I don't know if we ever explained what a server-side endpoint is compared to a client-side endpoint, but it might be confusing without the context. Maybe we can say that client-side endpoints delegate the evaluation of the toggle to the server while server-side does the evaluation?

Reply:
It's in the Unleash documentation, isn't it? This SDK can't be a replacement for all of documentation. Maybe adding a link, but that's it, IMO.

Reply:
Maybe linking to the documentation can help